modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -117,9 +147,25 @@ static PyObject* extract_proto_args(PyObject* py_args, PyObject* kw)
         {
             args.emplace_back(reinterpret_cast<pyopencv_GMat_t*>(item)->v);
         }
+        else if (PyObject_TypeCheck(item, reinterpret_cast<PyTypeObject*>(pyopencv_GArrayP2f_TypePtr)))
```

Comment:
Since each instance for `GArray` and `GOpaque` is a separate object we need to have a huge `if` here, we can make a separate object for Python that will contain all types for `GArray` and `GOpaque` and will be created like this:    
 ```g_in = GArray(cv.detail.OpaqueKind.RECT)```.
 I didn't do this because OpaqueKind is located in the internal namespace detail, let's discuss it locally

Reply:
Not relevant